gce_workload_cert_refresh/main.go

```diff
@@ -77,6 +77,10 @@ func getMetadata(key string) ([]byte, error) {
 		return nil, fmt.Errorf("HTTP 404")
 	}
 
+	if res.StatusCode == 412 {
```

Comment:
since this is an abuse of this code, let's add a comment about why we're treating it specially and when to expect it.